app/subscriber/src/features/my-reports/edit/content/ContentEditForm.tsx

```diff
@@ -27,17 +30,67 @@ export interface IContentEditFormProps {
 export const ContentEditForm = React.forwardRef<HTMLDivElement | null, IContentEditFormProps>(
   ({ disabled }, ref) => {
     const [{ userInfo }] = useApp();
+    const [{ myTonePool, init }, { storeMyTonePool }] = useProfileStore();
     const [, { updateReport }] = useReports();
     const [, { addContent, updateContent }] = useContent();
+    const [, { addMyTonePool, getMyTonePool }] = useTonePool();
     const { values, onNavigate, isSubmitting, setSubmitting, setValues, activeRow, setActiveRow } =
       useReportEditContext();
 
     const [form, setForm] = React.useState<IReportInstanceContentForm | undefined>(activeRow);
     const [errors, setErrors] = React.useState<IContentValidationErrors>();
+    const [userFormTonePool, setUserFormTonePool] = React.useState<
+      IContentTonePoolModel | undefined
+    >(undefined);
 
     const instance = values.instances.length ? values.instances[0] : undefined;
     const userId = userInfo?.id ?? 0;
 
+    React.useEffect(() => {
+      const getTonePool = async () => {
+        try {
+          if (!init.myTonePool && userId !== 0) {
+            const response = await getMyTonePool(userId);
```

Comment:
You are using redux to store their tonepool, yet you are going to make a request every time hit this page to get their tonepool from the API.